<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Backup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="source_label">
          <property name="text">
           <string>Source directory</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="source_path"/>
          </item>
          <item>
           <widget class="QPushButton" name="source_browse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="source_view">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="source_size_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="destination_label">
          <property name="text">
           <string>Destination directory</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="destination_path"/>
          </item>
          <item>
           <widget class="QPushButton" name="destination_browse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="destination_view">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="destination_size_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="check_file_contents">
          <property name="text">
           <string>Check file contents (for files of the same size)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_file_contents_newer">
          <property name="text">
           <string>But only check those with newer dates</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_bakcup_overwritten">
          <property name="text">
           <string>Backup overwritten files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scan_button">
          <property name="text">
           <string>Scan files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copy_button">
          <property name="text">
           <string>Copy new files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="do_remove_deleted">
          <property name="text">
           <string>Remove deleted files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progress_bar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="info_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
